css/src/new-redirects.css

```diff
@@ -81,6 +97,49 @@ body.seo_page_wpseo_redirects {
 		}
 	}
 
+	.yst-table--minimal table  {
+		.yst-table-header {
+			@apply yst-font-medium yst-text-slate-800;
+		}
+
+		.yst-table-cell {
+			@apply yst-pt-5 yst-pb-5 yst-text-slate-500;
+			.yst-checkbox__label {
+				margin-inline-start: 1.5rem;
+			}
+			.yst-text-field > div:first-child {
+				display: none !important;
+			}
+		}
+
+		.yst-checkbox__label {
+			margin-inline-start: 1rem;
```

Comment:
This can be replaced with `yst-ms-6`, maybe you would like to add it directly to the element.